environment.yml

```
@@ -1,11 +1,15 @@
-name: workshop
+name: image-analysis-23
 channels:
   - conda-forge
 dependencies:
+  - python=3.10
+  - pip
   - jupyterlab
   - jupytext
```

```suggestion
  - jupytext
  - notebook
```
Provide notebooks as an option as well?

---

Very good point! I thought notebook would get pulled down with jupytext but it does not! 🤯